components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -352,7 +355,7 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
 
         const client = await this.workspaceManagerClientProvider.get(runningInstance.region);
         const desc = await client.describeWorkspace(ctx, req);
-        const duration = desc.getStatus()!.getSpec()!.getTimeout() as WorkspaceTimeoutDuration;
+        const duration = this.userService.durationToWorkspaceTimeout(desc.getStatus()!.getSpec()!.getTimeout());
```

Comment:
Micro-nit: Same here, we're calling `somethingElseToTimeout(...getTimeout())` 🙈 (as in, turn a timeout into a timeout?)